server/service.go

```diff
@@ -21,7 +21,10 @@ var (
 )
 
 var (
-	errNoBlockHash = errors.New("no blockhash provided")
+	errInvalidHash      = errors.New("invalid blockhash provided")
```

Comment:
Should the rpc errors implement error codes as per spec?

Reply:
yes! will update codes in the next pr

Reply:
See #108